lib/backend.js

```diff
@@ -287,6 +287,35 @@ exports.createServer = (dynamodb, docClient) => {
     })
   )
 
+  app.delete('/api/deleteAllTables', asyncMiddleware(async (req, res) => {
```

Comment:
Not an issue but this is not very consistent route.
Maybe just `/tables` for deleting all and `/tables-prune` for pruning